src/app/about-us/page.tsx

```diff
@@ -1,22 +1,27 @@
 import React, { FC } from "react";
 import AboutHero from "@/components/about-us/AboutHero";
-import AboutOurMission from "@/components/about-us/AboutOurMission";
-import BoardMembers from "@/components/about-us/BoardMembers";
-import MissionStatement from "@/components/about-us/MissionStatement";
-import Timeline from "@/components/about-us/Timeline";
+import AboutOurMissionVision from "@/components/about-us/AboutOurMissionVision";
 import missionStatementData from "@/data/about-us";
-import PageHead from "@/components/about-us/PageHeader";
+import { Separator } from "@radix-ui/themes";
+import TimelineItem from "@/components/about-us/Timeline";
 
 const AboutUsPage: FC = () => {
-  const { missionStatement, aboutOurMission, timeline } = missionStatementData;
+  const { aboutOurMission, aboutOurVision, timeline } = missionStatementData;
 
   return (
-    <main className="">
+    <main>
       <AboutHero />
-      <MissionStatement missionStatement={missionStatement} />
-      <BoardMembers />
-      <AboutOurMission content={aboutOurMission} />
-      <Timeline items={timeline} />
+      <div className="flex mt-10 md:mt-20 bg-gray-50 h-full ">
```

Comment:
Suggestion (non-blocking): Would be good to use RadixUI's [Flex Component](https://www.radix-ui.com/themes/docs/components/flex) if possible.  It includes properties for margin.

**suggestion (non-blocking):** Would be good to use RadixUI's [Theme Component](https://www.radix-ui.com/themes/docs/theme/overview) if possible.  It lets you set text colors, background colors, and other properties.  Can use multiple on a page, nest themes, etc.